demos/demo37_experimental_mcmc_module_Gibbs.ipynb

```diff
@@ -117,11 +117,17 @@
     "    \"s\" : ConjugateNew()\n",
     "}\n",
     "\n",
-    "sampler_NUTS = GibbsNew(target, sampling_strategy_NUTS)\n",
+    "# Here we do 10 internal steps with NUTS for each Gibbs step\n",
+    "sampling_steps_NUTS = {\n",
```

Comment:
Great that you have implemented the possibility to configure the number of steps.  I'm a bit confused about how this works, maybe it is just the naming of the variables here.
1. Is it not possible to provide this as part of the sampling strategy?
2. Why it is called `sampling_steps_NUTS`, when it provides numbers of steps for both the NUTS part and the Conjugate part? Maybe just to be consistent with `sampler_NUTS` and `sampling_strategy_NUTS`, where the variable naming is just to distinguish from the previous cases of the notebook? This would make sense but still a bit confusing.

Reply:
Good question.

1) It can be viewed as "Intrinsic" to this Hybrid Gibbs sampling strategy. It is how many internal steps of each sampler we take in each Gibbs step. Each of the sub-samplers could technically contain this information, but it would only be used for Gibbs sampling. So I decided it was best to keep it part of the Gibbs class.

Reply:
2. Yes it was just to distinguish. I removed the postfix. Hope the notebook is more clear now.